File: app/src/main/java/easv/oe/dicecup2/dice/DiceHistoryManager.java
package easv.oe.dicecup2.dice;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

/**
 * stores dice rolls
 */
public class DiceHistoryManager implements Serializable {

    public List<DiceRollLog> historyList = new ArrayList<>();

    public List<DiceRollLog> getHistoryList() {
        return historyList;
    }


    public DiceHistoryManager() {
    }

    public void addToHistory(DiceRollLog history){
       historyList.add(history);
    }


}

File: app/src/main/java/easv/oe/dicecup2/dice/DiceHistoryManager.kt
package easv.oe.dicecup2.dice

import java.io.Serializable

class DiceHistoryManager : Serializable {
    var historyList: MutableList<DiceRollLog> = ArrayList()

    fun addToHistory(history: DiceRollLog) {
        historyList.add(history)
    }

    fun getLastRoll(): DiceRollLog {
        return historyList[historyList.size-1]
    }

}